src/commands/api/add.ts

```diff
@@ -0,0 +1,54 @@
+import { CliUx, Command, Flags } from '@oclif/core'
+import { ApiClaim, ApiType } from '../../models/api'
+import { ApiClaimService } from '../../services/api-claim-service'
+import { BundleService } from '../../services/bundle-service'
+
+export default class Add extends Command {
+  static description = 'Adds an internal API claim to the specified MFE component'
+
+  static examples = [
+    '$ entando-bundle-cli api add mfe1 ms1-api --serviceId ms1 --serviceUrl http://localhost:8080',
```

Comment:
`$ entando-bundle-cli api add` can be replaced with placeholder `<%= config.bin %> <%= command.id %>`